From 3b8bcdce7180aad9da67015b408d2a4bae54555d Mon Sep 17 00:00:00 2001
From: Gaurang Khodifad <gaurang.khodifad@volansystech.com>
Date: Tue, 9 Jan 2018 17:27:13 +0530
Subject: [PATCH] SPI configuration with 'spidev' compatible

Prepared this patch to avoid warning of buggy DT at start up.

Signed-off-by: Gaurang Khodifad <gaurang.khodifad@volansystech.com>
---
 drivers/spi/spidev.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/spi/spidev.c b/drivers/spi/spidev.c
index 2e05046..d780491 100644
--- a/drivers/spi/spidev.c
+++ b/drivers/spi/spidev.c
@@ -696,6 +696,7 @@ static int spidev_release(struct inode *inode, struct file *filp)
 static const struct of_device_id spidev_dt_ids[] = {
 	{ .compatible = "rohm,dh2228fv" },
 	{ .compatible = "lineartechnology,ltc2488" },
+	{ .compatible = "spidev" },
 	{},
 };
 MODULE_DEVICE_TABLE(of, spidev_dt_ids);
-- 
1.9.1

